packages/games/src/constants/games/karatGalaxy.ts

```diff
@@ -0,0 +1,80 @@
+import { GameType, GenreType, PostersItemDataType, PostersLayout, TrendingTagType } from '../../types'
+
+export const karatGalaxy: GameType = { 
+  id: 'karat-galaxy', 
+  projectName: 'Karat', 
+  title: 'Karat Galaxy', 
+  subTitle: 'Embark on a Cosmic Karat Quest: Shoot, mine, win!', 
+  description: 'Join an interstellar adventure where mining, NFTs, DeFi, and fun unite.  Power up with KARAT and turn CAKE tokens into gold coins, unlocking exciting in-game adventures. Enjoy exclusive bonuses with Pancake Squad and Bunny NFTs. Battle, mine, win, and conquer the cosmos.', 
+  publishDate: 1698044972, // Timestamp in seconds 
+ genre: GenreType.TowerDefense, 
+ trendingTags: [ 
+    TrendingTagType.Strategy 
+ ],
+ // 1600px x 224px  
+ headerImage: 'https://statics.karatgalaxy.io/pancakeswap-game-landing/header.jpg',
+ headerIconImage: { 
+  // 716px × 290px 
+    desktop: 'https://statics.karatgalaxy.io/pancakeswap-game-landing/header-icon.png',
+  // 215px × 174px 
+    mobile: 'https://statics.karatgalaxy.io/pancakeswap-game-landing/header-icon.png'
+  }, 
+  projectLogo: { 
+  // 153px × 55px 
+    lightTheme: 'https://statics.karatgalaxy.io/pancakeswap-game-landing/logo.svg', 
+  // 153px × 55px 
+    darkTheme: 'https://statics.karatgalaxy.io/pancakeswap-game-landing/logo.svg', 
+  }, 
+  projectCircleLogo: { 
+  // 120px × 120px 
+    lightTheme: 'https://statics.karatgalaxy.io/pancakeswap-game-landing/circle-logo.png',
+    // 120px × 120px 
+	darkTheme: 'https://statics.karatgalaxy.io/pancakeswap-game-landing/circle-logo.png',
+  }, 
+  gameLink: 'https://statics.karatgalaxy.io/', 
```

Comment:
Incorrect game link type. Please follow the type definition here
https://github.com/pancakeswap/pancake-frontend/blob/develop/packages/games/src/types.ts#L53-L57